### PR TITLE
implement special symbols.

### PR DESCRIPTION
**Pull request by oscarlevin**
Added a component for all the special symbols in the `special-symbols.xml` fragment

### CODE REVIEW

File: packages/jsx/src/target/html/components/special-symbols.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+};
+
+export const SpecialSymbol: ReplacerComponent = function ({ node }) {
+
+    const symbolName = node.name;
+    const symbol = SYMBOL_MAP[symbolName] || symbolName;
+    if (symbolName === "times") {
+      return (<span className="times-sign">×</span>);
+    }
+    return (symbol);
```
Comment (siefkenj) [inline packages/jsx/src/target/html/components/special-symbols.tsx:31]: Can you run prettier on this file?

File: packages/fragment/tests/fragments/special-symbols-1.xml [hunk 2]
```diff
@@ -1,19 +1,2 @@
...
-    <pilcrow />
-    <section-mark />
-    <minus/>
-    <times/>
-    <solidus/>
-    <obelus/>
-    <plusminus/>
-</FRAGMENT>
\ No newline at end of file
+    <copyright /> is a copyright, while <phonomark /> is a phonomar. <copyleft /> is the copyleft symbol. <registered /> is a registered trademark. <trademark /> is a trademark. <servicemark /> is a servicemark. <ellipsis /> are ellipsis. And <midpoint /> is a midpoint. <swungdash /> is called a swungdash. <permille /> is a permille. <pilcrow /> is a pilcrow. Other symbols are <section-mark />, <minus />, <times />, <solidus />, and <obelus />. Finally, <plusminus /> is a plus-minus sign. </FRAGMENT>
```
Comment (siefkenj) [inline packages/fragment/tests/fragments/special-symbols-1.xml:2]: This is very strange formatting...can your auto-formatter do better? Or at least put the closing `</FRAGMENT>` tag on a different line?

Comment (oscarlevin) [reply packages/fragment/tests/fragments/special-symbols-1.xml:2]: The XML formatter was what gave me this.  I don't understand why that is so messed up.  I'll fix though.

File: packages/jsx/src/target/html/components/special-symbols.tsx (hunk 1 above)
Comment (oscarlevin) [reply packages/jsx/src/target/html/components/special-symbols.tsx]: Should be fixed now.

**Comment by siefkenj** [comment]
Thanks!

**Pull request merged**